packages/test/test-gc-sweep-tests/src/testDataObjects.ts

```diff
@@ -30,7 +30,7 @@ import {
     SharedMapHandler,
 } from "./channelHandler";
 
-// data store that exposes container runtime for testing.
+// The BaseTestDataObject class that exposes useful objects for testing
```

Comment:
This comment does not convey anything meaningful. It can probably just say that this is the base data object that has basic functionality from which other data objects should derive.